From df92917582d785255cbf4b3bb13c24c7d1609993 Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Sat, 9 Jun 2018 04:04:00 -0700
Subject: [PATCH] MANATEE-398 manatee-adm history -j doesn't understand v1
 timestamps Reviewed by: Richard Bradley <richard.bradley@joyent.com> Approved
 by: Richard Bradley <richard.bradley@joyent.com>

---
 bin/manatee-adm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/manatee-adm b/bin/manatee-adm
index 7e34493..13dbd22 100755
--- a/bin/manatee-adm
+++ b/bin/manatee-adm
@@ -719,7 +719,7 @@ ManateeAdm.prototype.do_history = function (subcmd, opts, args, cb) {
 
             if (evt.action) {
                 return ({
-                    'TIME': evt.date.toISOString(),
+                    'TIME': evt.time.toISOString(),
                     'G#': '-',
                     'MODE': '-',
                     'FRZ': '-',
-- 
2.21.0

